tests/test_model_layers.py

```diff
@@ -513,6 +513,24 @@ def test_FunctionalChainInvolvementLink_attributes(
     link_uuid: str,
     target_uuid: str,
 ) -> None:
+    """
+    Test if a FunctionalChainLink covers all of its attributes.
+
+    Covered attributes are
+        * exchanged_items - List of all ExchangeItems spec. on the
+          FunctionalChainLink
+        * exchange_context - A context markup.
```

Comment:
Do we really want this information duplicated here? IMO this part of the docstring doesn't add any value, but does add maintenance overhead. We could perhaps make it more clear from the code by restructuring it a bit though.

Reply:
You are right. No we don't.